### ISSUE TITLE
[a11y]: CheckboxGroup has ARIA attribute is not allowed: aria-readonly="false"

### ISSUE THREAD

**Issue by meenaid114-cloud** [carbon-design-system/carbon#21025]
### Package

@carbon/react

### Browser

Chrome

### Operating System

Windows

### Package version

1.93.0

### React version

18.3.1

### Automated testing tool and ruleset

axe-core/playwright

### Description

Ensure an element's role supports its ARIA attributes
<html>
<body>

Element locationfieldsetElement source<fieldset class="cds--checkbox-group sfg--filter-field checkbox-group" aria-readonly="false"> | Fix all of the following:ARIA attribute is not allowed: aria-readonly="false"
-- | --

</body>
</html>

### WCAG 2.1 Violation

wcag412

### Reproduction/example

checkbox-group

### Steps to reproduce

checkbox-group in storybook, run accessibility checker. You may need to flip readonly to false in the playground.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Hey @meenaid114-cloud, thank you for your issue. I wasn’t able to reproduce it by following your steps. Could you please share some screenshots, just so we can confirm whether the issue still exists or if it has already been fixed? Thank you very much.

**Comment by heloiselui** [comment]
Hi, I’m going to close this issue since I wasn’t able to reproduce it, but if it’s still happening on your side, we can reopen it or open a new one with more details and a reproduction. Thanks!!

**Comment by meenaid114-cloud** [comment]
<img width="1151" height="721" alt="Image" src="https://github.com/user-attachments/assets/191b2cd7-441c-413a-b9dd-c9dc5c21c09c" />

<img width="655" height="248" alt="Image" src="https://github.com/user-attachments/assets/e0e7975f-14fd-43b1-a71f-c4809f44476f" />
im sharing the screen shot of  code checkbox group
